Recommendo-api/Startup.cs

```diff
@@ -21,23 +22,29 @@ public Startup(IConfiguration configuration)
         // This method gets called by the runtime. Use this method to add services to the container.
         public void ConfigureServices(IServiceCollection services)
         {
-            var connection = Configuration.GetConnectionString("recommendo-recipeDatabase");
-            services.AddDbContextPool<RecommendoRecipesContext>(options => options.UseSqlServer(connection));
-            services.AddControllers();
             services.AddCors(options =>
             {
-                options.AddPolicy(name: "allowed",
+                options.AddPolicy(name: MyAllowSpecificOrigins,
```

Comment:
did you just open it up to all?

Reply:
yeah don't need to actually, just so we can change stuff